packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -918,14 +951,27 @@ const useAzureCommunicationCallAdapterGeneric = <
           adapterRef.current = undefined;
         }
 
-        let newAdapter = await createAzureCommunicationCallAdapter({
-          credential,
-          displayName,
-          locator,
-          userId,
-          /* @conditional-compile-remove(PSTN-calls) */ alternateCallerId,
-          /* @conditional-compile-remove(rooms) */ options
-        });
+        let newAdapter: AdapterType;
+        if (type === 'AzureCommunication') {
+          if (!('displayName' in args) || !args?.displayName) {
+            return;
```

Comment:
we already checked for displayName on L937, I assume this check is to satiate typescript, if so can this be throw new Error instead of return as this code should be unreachable